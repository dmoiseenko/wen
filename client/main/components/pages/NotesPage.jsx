import React from 'react';

import NotesContainer from '../notes/Notes.container';
import NoteFormContainer from '../noteForm/NoteForm.container';


export default () => (
  <div className="container">
    <div className="content">
      <h2>Add New Note</h2>
    </div>
    <NotesContainer />
    <NoteFormContainer />
  </div>
);
