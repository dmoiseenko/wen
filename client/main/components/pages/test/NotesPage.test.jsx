import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import NotesPage from '../NotesPage';


it('should render', () => {
  const wrapper = shallow(
    <NotesPage />
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});
