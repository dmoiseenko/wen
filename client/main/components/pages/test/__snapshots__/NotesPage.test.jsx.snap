// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="container"
>
  <div
    className="content"
  >
    <h2>
      Add New Note
    </h2>
  </div>
  <Connect(Apollo(lifecycle(mapProps(Notes)))) />
  <Connect(ReduxForm) />
</div>
`;
